src/components/ListItem.js

```diff
@@ -0,0 +1,18 @@
+import { useProducts } from '../hooks/useProducts';
+
+const ListItem = () => {
+  const { products, loading, error } = useProducts();
+
+  return (
+    <div>
+      {loading && 'Loading'}
```

Comment:
In general, a comparison { something && result } is OK. However, there are some cases where you can get unexpected results.

For example: if an error is null, according to Javascript: the result could be falsy, then React renders nothing. In other cases, when loading is 1 (true) or 0 (false) in some cases depends on your condition.

Some people suggested that it's much better you use an explicit condition like:
{ loading ? <p>'Loading'... </p>: null }
{error !== null ? <p>There was an error.</p>: null} or {error !== null ? <p>{error.message}</p>: null}

**^^They are only suggestions, not blocking. 😃**